### PR TITLE
start work to implement spim_fit_control for thinned trajectories

### PR DESCRIPTION
**Pull request by epiyasin**
Allows for ncov https://github.com/ncov-ic/ncov-outputs/pull/2064 to be able to use the thinned trajectories from https://github.com/mrc-ide/sircovid/pull/388 by removing downstream dependencies in fitting and simulation task of outdated/legacy fitting/simulating/plotting tasks.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: spimalot
 Title: Support for 'sircovid'
-Version: 0.6.12
+Version: 0.6.13
```
Comment (richfitz) [inline DESCRIPTION:3]: Do you need your changes from sircovid here? If so then increase the required version number below. If not then we will merge this before the sircovid PR

**Review by richfitz — CHANGES_REQUESTED** [review]
No complaints about the implementation, just the sequencing

File: DESCRIPTION (hunk 1 above)
Comment (epiyasin) [reply DESCRIPTION:3]: Yes they are needed here as the size of the output changes and spimalot expects a new length of state trajectories produced from sircovid. Have bumped the version up now thank you.

**Review by edknock — APPROVED** [review]
Looks good to me

**Review by richfitz — APPROVED** [review]
Always nice to see old code get pulled out

**Pull request merged**